src/main/resources/assets/computercraft/lua/rom/programs/mkdir.lua

```diff
@@ -5,5 +5,11 @@ if #tArgs < 1 then
 end
 
 local sNewDir = shell.resolve( tArgs[1] )
+
+if fs.exists( sNewDir ) == true then
```

Comment:
This check is unnecessary. Making a dir that already exists should just silently suceed.

Reply:
Yes, when it is a Directory. But if it is a normal file, it will give a lua error.